model/registry/geo.yaml

```diff
@@ -0,0 +1,114 @@
+groups:
+  - id: registry.geo
+    type: attribute_group
+    prefix: geo
+    brief: >
+      Geo fields can carry data about a specific location related to an event.
+      This geolocation information can be derived from techniques such as Geo IP, or be user-supplied.
+    attributes:
+      - id: city_name
+        stability: experimental
+        type: string
+        brief: >
+          City name.
+        examples: [ 'Montreal', 'Berlin' ]
+      - id: continent_code
+        stability: experimental
+        brief: >
+          Two-letter code representing continent’s name.
+        type:
+          allow_custom_values: false
```

Comment:
we don't have restrictions on custom values anymore and this check will be removed. 
Although in this case it makes non difference :)